packages/amplify-provider-awscloudformation/src/index.ts

```diff
@@ -159,6 +161,7 @@ module.exports = {
   createDynamoDBService,
   resolveAppId,
   loadConfigurationForEnv,
+  getAccountId,
```

Comment:
I don't think we need to export this twice.

Reply:
Both exports were required when I called it via 

```
const awsConfigInfo = await context.amplify.invokePluginMethod(
  context,
  "awscloudformation",
  undefined,
  "getAccountId",
  [context, environmentName, appId]
);
```

But with `context.amplify.executeProviderUtils` I don't think it's necessary anymore.